README.md

```diff
@@ -4,4 +4,6 @@ This plugin wraps [ArcWelderLib](https://github.com/FormerLurker/ArcWelderLib) b
 
 Inspired by on a postprocessing script by [yysh12](https://github.com/yysh12)
 
-For further informatio about ArcWelder antistutter, see https://plugins.octoprint.org/plugins/arc_welder
\ No newline at end of file
+For further informations about ArcWelder antistutter, see https://plugins.octoprint.org/plugins/arc_welder
+
+For further informations about MM Per Arc Segment and Min Arc Segments options, see https://github.com/FormerLurker/ArcWelderPlugin/issues/80
```

Comment:
I'm not sure if we should link to specific issues for specific features. Ideally, the description of the setting would be good enough, and eventually I expect any documentation in that issue to trickle down to the official documentation of the arcwelder plugin.